app/components/edit-reminder.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  store: Ember.inject.service(),
+  router: Ember.inject.service('-routing'),
```

Comment:
This is interesting. I'm not sure how I feel about this. I believe re-routing should be passed to the controller or preferably the router.